admin_guide/install/first_steps.adoc

```diff
@@ -0,0 +1,77 @@
+= First Steps
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+These steps will help you populate your OpenShift deployment with a useful set of `ImageStreams` and `Templates` that will make it easy for developers to create new applications.  The instructions will create these objects in the `openshift` project which is a default project to which all users have view access.
+
+== Prerequisites
+* You have deployed the OpenShift Docker registry service
+* You are able to run client commands with cluster-admin authority
+* You have cluster-admin privileges.  You will need to run all of these commands as cluster-admin because they operate on the `openshift` project.
+
+== Create ImageStreams for OpenShift Images
+The core set of image streams provide images that can be used to build NodeJS, Perl, PHP, Python, and Ruby applications.  It also defines images for MongoDB, MySQL,and PostgreSQL to support data storage.
+
+To create the core set of image streams, run:
+
+    $ oc create -f examples/image-streams/image-streams-centos7.json -n openshift
+
+*Note:* if you are setting up a subscribed system and prefer to use the RHEL7 based images,  run this command instead:
+    
+    $ oc create -f examples/image-streams/image-streams-rhel7.json -n openshift
+
+It is not possible to create both sets of ImageStreams because they use the same name.  If you desite to have both sets of ImageStreams available to users, either create one set in a different Project, or edit one of the files and modify the ImageStream names to make them unique.
+
+== Create ImageStreams for JBoss Middleware Images
+The JBoss Middleware image streams provide images for JBoss EAP, JBoss EWS, and JBoss AMQ.  They can be used to build applications for those platforms or run directly as is.
+
+    $ oc create -f examples/middleware-streams/jboss-image-streams.json
```

Comment:
Actual base path would be /usr/share/openshift/examples but that's pretty long, we could tell them to cd /usr/share/openshift first and then keep the commands as is

+1 on examples/image-streams and examples/templates
Any reason we'd want to keep middleware-streams and db-templates separate, couldn't they all go into the same directories provided there's no filename conflicts?